projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py

```diff
@@ -46,7 +49,7 @@ def __init__(self, input_image_topic="/usb_cam/image_raw", input_depth_image_top
 
         self.gesture_publisher = rospy.Publisher(gesture_annotations_topic, Classification2D, queue_size=10)
 
-        image_sub = message_filters.Subscriber(input_image_topic, ROS_Image)
+        image_sub = message_filters.Subscriber(input_rgb_image_topic, ROS_Image)
         depth_sub = message_filters.Subscriber(input_depth_image_topic, ROS_Image)
```

Comment:
I think we should modify the queue size to avoid delays relating to #275.

```suggestion
        image_sub = message_filters.Subscriber(input_rgb_image_topic, ROS_Image, queue_size=1, buff_size=10000000)
        depth_sub = message_filters.Subscriber(input_depth_image_topic, ROS_Image, queue_size=1, buff_size=10000000)
```